vision/product_search/create_product_set.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package productsearch
+
+// [START vision_product_search_create_product_set]
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	vision "cloud.google.com/go/vision/apiv1"
+	visionpb "google.golang.org/genproto/googleapis/cloud/vision/v1"
+)
+
+func createProductSet(w io.Writer, projectID string, location string, productSetID string, productSetDisplayName string) error {
+	ctx := context.Background()
+	c, err := vision.NewProductSearchClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	req := &visionpb.CreateProductSetRequest{
+		Parent: fmt.Sprintf("projects/%s/locations/%s", projectID, location),
+		ProductSetId: productSetID,
+		ProductSet: &visionpb.ProductSet{
+			DisplayName: productSetDisplayName,
+		},
+	}
+
+	resp, err := c.CreateProductSet(ctx, req)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintln(w, "Product set name:", resp.Name)
```

Comment:
```suggestion
	fmt.Fprintf(w, "Product set name: %s", resp.Name)
```

Can add `\n` at the end if you prefer.